serialport.go

```diff
@@ -273,7 +274,11 @@ func (p *serport) writerRaw() {
 	h.broadcastSys <- []byte(msgstr)
 }
 
+var spHandlerLock sync.Mutex
```

Comment:
What is the resource protected by this mutex? I would add a comment a about that